x-pack/plugins/secops/public/components/notes/translations.ts

```diff
@@ -15,21 +15,33 @@ export const USER = i18n.translate('xpack.secops.notes.user', {
 });
 
 export const ADD_NOTE = i18n.translate('xpack.secops.notes.addNote', {
-  defaultMessage: 'Add note',
+  defaultMessage: 'Add Note',
```

Comment:
The keys for these `i18n` should follow the guidelines:
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

So I think that this should end with at least a `Description` as in `addNoteDescription` and the rest below would follow at least one of those ending patterns.

Reply:
Thanks, I used vscode's `Find All References` action on each `export`ed variable in this `translations` file, and updated all entries to include the "context as a suffix", per the guidelines.